modules/oadp-ceph-cephfs-back-up.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+ifeval::["{context}" == "cephfs"]
+:cephfs:
+endif::[]
+ifeval::["{context}" == "split"]
+:split:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="oadp-ceph-cephfs-back-up_{context}"]
+
+ifdef::cephfs[]
+= Backing up data using OADP 1.2 Data Mover and CephFS storage
+You can use OpenShift API for Data Protection (OADP) 1.2 Data Mover to back up data using CephFS storage by enabling the shallow copy feature of CephFS storage.
+endif::cephfs[]
+
+ifdef::split[]
+= Backing up data using OADP 1.2 Data Mover and split volumes
+You can use OpenShift API for Data Protection (OADP) 1.2 Data Mover to back up data in an environment that has split volumes.
+endif::split[]
+
+.Procedure
+
+. Create a `Backup` CR similar to the following:
++
+.Example `Backup` CR
++
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: <backup-name>
+  namespace: <protected-ns>
+spec:
+  includedNamespaces:
+  - <app-ns>
+  storageLocation: velero-sample-1
+----
+
+. Monitor the progress of the `VolumeSnapshotBackup` CRs by doing the following:
```

Comment:
Just a nit from the [ISG guidance](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#introducing-the-procedure) on lead-in sentences: 
```suggestion
. Monitor the progress of the `VolumeSnapshotBackup` CRs by completing the following steps:
```